medicines/api/src/product.rs

```diff
@@ -1,9 +1,10 @@
-use crate::{document::Document, substance::Substance};
-use juniper::GraphQLObject;
+use crate::{
+    document::{self, get_documents_graph_from_documents_vector, Document},
+    substance::Substance,
+};
 use search_client::{models::IndexResult, Search};
 
-#[derive(GraphQLObject, Debug, Eq, Ord, PartialEq, PartialOrd)]
-#[graphql(description = "A medical product containing active ingredients")]
+#[derive(Debug, Eq, Ord, PartialEq, PartialOrd)]
 pub struct Product {
     name: String,
     document_count: i32,
```

Comment:
Why do we have document count here? Should be removed in favour of `totalCount` inside documents.

Reply:
the struct is no longer used to populate the graphql object – it's one of the nice advantages of using `#[graphql_object]` that we can decouple the type in rust from the fields sent to the user